developing_rpkgs.qmd

```diff
@@ -162,14 +167,37 @@ R code is documented using [{roxygen2}](https://roxygen2.r-lib.org/). Here is an
 
 Using these examples, add documentation to the functions in your R package. Note that the `@export` tag should be used if you want users to be able to access the function directly.
 
```

Comment:
Cool, this is new to me! Which IDEs have this shortcut? RStudio and VScode? Or just one of them?

Reply:
If you could add this detail in the main branch, that would be great :)

Reply:
The shortcut to get the roxygen2 skeleton? It's a shortcut in the rstudio IDE. Honestly if I were to develop a package from scratch, I'd probably start in the rstudio IDE and then switch to vscode after bulk of it is created. I like the shortcuts and built-in features to actually create a package of rstudio but I like the git/github features in vscode better for pulling down a package and making edits.